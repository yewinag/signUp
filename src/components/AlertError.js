import React, { Component } from 'react'
import { Text, View } from 'react-native'

import alertError from '../styles/alert-error';


const AlertError = (props) => (
    <View>
        <Text style={alertError.text}>{props.err}</Text>
    </View>
)
export default AlertError
