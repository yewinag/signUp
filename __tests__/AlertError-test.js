import 'react-native';
import React from 'react';
import AlertError from '../src/components/AlertError';

import renderer from 'react-test-renderer';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';

Enzyme.configure({ adapter: new Adapter() });

it('test render AlertError correctly', () => {
  const tree = renderer.create(<AlertError />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('test AlertError props', ()=> {
    const wrapper = shallow(<AlertError err="Enter is repuried" />);
    expect((wrapper.props().children.props.children)).toEqual("Enter is repuried")
})