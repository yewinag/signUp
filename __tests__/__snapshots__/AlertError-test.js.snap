// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#ff0033",
      }
    }
  />
</View>
`;

exports[`test render AlertError correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#ff0033",
      }
    }
  />
</View>
`;
